src/content/app-platform/defaulting-validation/index.md

```diff
@@ -23,7 +23,7 @@ user_questions:
 
 For Giant Swarm releases using app-operator version 3.0.0 and upwards the
 defaulting and validation logic of [App CRs]({{< relref "/ui-api/management-api/crd/apps.application.giantswarm.io.md" >}}) is enabled.
-As of writing this is Azure releases from v13.1.0 onwards. AWS and KVM support will be added in upcoming releases.
+This is AWS releases from v14.0.0 onwards and Azure and KVM releases from v13.1.0 onwards.
```

Comment:
```suggestion
Following releases will start validation by app-admission-controller. 
- AWS: >= v14.0.0 
- Azure: >= v13.1.0 
- KVM: >= v13.1.0 
```

Listing releases is more comfortable I think. WDYT?

Reply:
Thanks! Yes I think its better as a list. I just changed the text a bit as the next paragraph mentions app-admission-controller. PTAL